code/_onclick/hud/alert.dm

```diff
@@ -877,6 +877,11 @@ or shoot a gun to move around via Newton's 3rd Law of Motion."
 	carbon_owner.changeNext_move(CLICK_CD_RESIST)
 	carbon_owner.shoes.handle_tying(carbon_owner)
 
+/atom/movable/screen/alert/iv_connected
+	name = "IV Connected"
+	desc = "You have an IV connected to your arm. Remember to remove it or drag the IV stand with you before moving, or else it will rip out!"
+	icon_state = ALERT_IV_CONNECTED
```

Comment:
I think this should exist just in the `iv_drip` file, then you can have `ALERT_IV_CONNECTED` only exist in that file

Reply:
Sure